typings/tsd.d.ts

```diff
@@ -1,7 +1,7 @@
-/// <reference path="../node_modules/immutable/dist/immutable.d.ts" />
-/// <reference path="../node_modules/reflect-metadata/reflect-metadata.d.ts" />
 /// <reference path="chrome/chrome.d.ts" />
 /// <reference path="es6-promise/es6-promise.d.ts" />
 /// <reference path="filesystem/filesystem.d.ts" />
 /// <reference path="filewriter/filewriter.d.ts" />
 /// <reference path="webrtc/MediaStream.d.ts" />
+/// <reference path="../node_modules/immutable/dist/immutable.d.ts" />
```

Comment:
Do we need these? Shouldn't these be picked up from `node_modules` automatically?